WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -350,6 +351,7 @@ namespace Plugin {
     void WebKitBrowser::Deactivated(RPC::IRemoteConnection* connection)
     {
         if (connection->Id() == _connectionId) {
+            TRACE(Trace::Error, (_T("WebKitBrowser::Deactivated: { \"URL\": %s }"), _lastURL.substr(0,80).c_str()));
```

Comment:
I think You can avoid a mem copy with '%.*s' format and specifying limit just before _lastURL.c_str(). i.e.:

```
TRACE(Trace::Error, (_T("WebKitBrowser::Deactivated: { \"URL\": %.*s }"), 80, _lastURL.c_str()));
```

could you please give it a try?

Reply:
@emutavchi , the above format is working fine.